action_manager.py

```diff
@@ -128,8 +128,11 @@ def pickup_action(cls):
         """
         loot = [e for e in cls.dungeon.current_level.entities if e.pos == cls.player.pos and e != cls.player]
 
-        if loot:
-            cls.player.backpack.add(loot[0], 1)
-            cls.dungeon.current_level.entities.remove(loot[0])
-            return True
+        while loot:
+            # Only attempt to pick up entities of type item
+            if loot[0].type == 'item':
```

Comment:
or just replace `e != cls.player` in the list comprehension by `e.type == 'item'`

Reply:
true